pyccel/ast/literals.py

```diff
@@ -217,3 +217,12 @@ def get_default_literal_value(dtype):
         raise TypeError('Unknown type')
     return value
 
+class Nil(Basic):
+
+    """
+    class for None object in the code.
+    """
+
+    def _sympystr(self, printer):
```

Comment:
Can this be changed to:
```
def __str__(self):
    return 'None'
```
Or does that break something?

Reply:
i will check